packages/reactive-dict/reactive-dict.js

```diff
@@ -36,25 +36,40 @@ ReactiveDict = function (dictName) {
 };
 
 _.extend(ReactiveDict.prototype, {
-  set: function (key, value) {
+  // set() began as a key/value method, but we are now overloading it
+  // to take an object of key/value pairs, similar to backbone
+  // http://backbonejs.org/#Model-set
+
+  set: function (key_or_object, value) {
     var self = this;
 
-    value = stringify(value);
+    // if the first argument is a string, use the original key/value method
+    if (typeof key_or_object === 'string') {
```

Comment:
To avoid wrapping this whole block in an if, let's just have a short-circuit return in the front:

```
if ((typeof key_or_object === 'object') && (value === undefined)) {
   self._setObject(key_or_object);
   return;
}

// Then the rest of the code
```